### ISSUE TITLE
opm doesn't support building index with v1 CRD

### ISSUE THREAD

**Issue by IBMRob** [operator-framework/operator-registry#293]
With the latest release of the operator-sdk its possible to create CRDs which use apiextensions.k8s.io/v1. When attempting to build a registry-index which includes these it fails to builds:
```
$ opm index add --container-tool docker --bundles default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0 --tag default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/company-product-index:1.0.0
INFO[0000] building the index                            bundles="[default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0]"
INFO[0000] running /usr/local/bin/docker pull default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0  bundles="[default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0]"
INFO[0003] running /usr/local/bin/docker pull default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0  bundles="[default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0]"
INFO[0005] running docker save                           bundles="[default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0]"
INFO[0005] found annotations file searching for csv      dir=bundle_tmp312951404 file=bundle_tmp312951404/metadata load=annotations
INFO[0005] found csv, loading bundle                     dir=bundle_tmp312951404 file=bundle_tmp312951404/manifests load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=product.company.com_configurations_crd.yaml load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=product.company.com_dashboards_crd.yaml load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=product.company.com_designerauthorings_crd.yaml load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=product.company.com_integrationservers_crd.yaml load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=product.company.com_switchservers_crd.yaml load=bundle
INFO[0005] loading bundle file                           dir=bundle_tmp312951404/manifests file=company-product-operator.clusterserviceversion.yaml load=bundle
ERRO[0005] permissive mode disabled                      bundles="[default-route-openshift-image-registry.apps.myServer.com/openshift-marketplace/product-operator-bundle:1.0.0]" error="error loading bundle from image: error checking provided apis in bundle company-product-operator.v1.0.0: error decoding CRD: no kind \"CustomResourceDefinition\" is registered for version \"apiextensions.k8s.io/v1\" in scheme \"github.com/operator-framework/operator-registry/pkg/registry/bundle.go:15\""
Error: error loading bundle from image: error checking provided apis in bundle company-product-operator.v1.0.0: error decoding CRD: no kind "CustomResourceDefinition" is registered for version "apiextensions.k8s.io/v1" in scheme "github.com/operator-framework/operator-registry/pkg/registry/bundle.go:15"
```

**Comment by dinhxuanvu** [comment]
Hey Rob, we are aware of this issue and there is work underway to enable opm to support CRD v1. Stay tuned. Thanks.

**Comment by arthurbarr** [comment]
Hi @dinhxuanvu , do you have any idea when you're targeting this work for?  It would be helpful to know if we should back-out our use of the v1 CRD API, or if support might land fairly soon.

**Comment by dinhxuanvu** [comment]
Hey @arthurbarr, there is a pending PR on the work and it is expected to merge by the end of this week.

**Comment by dinhxuanvu** [comment]
Hey folks, the latest release of registry/opm supports v1 CRD. There is only Mac opm binary attached in the release but you can build the opm binary yourself if you are on Linux. There is an issue with the build process at the moment.

**Comment by huzefa51** [comment]
Is this issue still not fixed?

time="2020-07-15T14:31:26Z" level=info msg="Using in-cluster kube client config"
time="2020-07-15T14:31:26Z" level=info msg="operator source(s) specified are - [https://quay.io/cnr|huzefa51]" port=50051 type=appregistry
time="2020-07-15T14:31:26Z" level=info msg="package(s) specified are - my-operator" port=50051 type=appregistry
time="2020-07-15T14:31:26Z" level=info msg="input has been sanitized" port=50051 type=appregistry
time="2020-07-15T14:31:26Z" level=info msg="sources: [https://quay.io/cnr/huzefa51]" port=50051 type=appregistry
time="2020-07-15T14:31:26Z" level=info msg="packages: [my-operator]" port=50051 type=appregistry
time="2020-07-15T14:31:27Z" level=info msg="the following releases are available for package my-operator -> [0.0.5 0.0.3 0.0.2 0.0.1]" port=50051 type=appregistry
time="2020-07-15T14:31:27Z" level=info msg="resolved the following packages: [my-operator:0.0.5]" port=50051 type=appregistry
time="2020-07-15T14:31:27Z" level=info msg="downloading repository: my-operator:0.0.5 from https://quay.io/cnr" port=50051 type=appregistry
time="2020-07-15T14:31:28Z" level=info msg="download complete - 1 repositories have been downloaded" port=50051 type=appregistry
time="2020-07-15T14:31:28Z" level=info msg="decoded 0 flattened and 1 nested operator manifest(s)" port=50051 type=appregistry
time="2020-07-15T14:31:28Z" level=info msg="loading Bundles" dir=downloaded
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=downloaded load=bundles
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=my-operator load=bundles
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=my-operator-lqlqunmc load=bundles
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=0.0.5 load=bundles
time="2020-07-15T14:31:28Z" level=info msg="found csv, loading bundle" dir=downloaded file=myapp-operator.v0.0.5.clusterserviceversion.yaml load=bundles
time="2020-07-15T14:31:28Z" level=info msg="loading bundle file" dir=downloaded/my-operator/my-operator-lqlqunmc/0.0.5 file=myapp-operator.v0.0.5.clusterserviceversion.yaml load=bundle
time="2020-07-15T14:31:28Z" level=info msg="loading bundle file" dir=downloaded/my-operator/my-operator-lqlqunmc/0.0.5 file=myapp.example.com_myapps_crd.yaml load=bundle
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=manifests load=bundles
time="2020-07-15T14:31:28Z" level=info msg="found csv, loading bundle" dir=downloaded file=myapp-operator.clusterserviceversion.yaml load=bundles
time="2020-07-15T14:31:28Z" level=info msg="loading bundle file" dir=downloaded/my-operator/my-operator-lqlqunmc/manifests file=myapp-operator.clusterserviceversion.yaml load=bundle
time="2020-07-15T14:31:28Z" level=info msg="loading bundle file" dir=downloaded/my-operator/my-operator-lqlqunmc/manifests file=myapp.example.com_myapps_crd.yaml load=bundle
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=metadata load=bundles
time="2020-07-15T14:31:28Z" level=info msg="loading Packages and Entries" dir=downloaded
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=downloaded load=package
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=my-operator load=package
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=my-operator-lqlqunmc load=package
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=0.0.5 load=package
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=manifests load=package
time="2020-07-15T14:31:28Z" level=info msg=directory dir=downloaded file=metadata load=package
**time="2020-07-15T14:31:28Z" level=warning msg="strict mode disabled" error="error loading manifests from appregistry: error loading operator manifests: [error checking provided apis in bundle : error decoding CRD: no kind \"CustomResourceDefinition\" is registered for version \"apiextensions.k8s.io/v1\" in scheme \"pkg/registry/bundle.go:15\", error adding operator bundle : error decoding CRD: no kind \"CustomResourceDefinition\" is registered for version \"apiextensions.k8s.io/v1\" in scheme \"pkg/registry/bundle.go:15\", error loading package into db: [FOREIGN KEY constraint failed, no default channel specified for myapp-operator], error loading package into db: [FOREIGN KEY constraint failed, no default channel specified for my-operator]]" port=50051 type=appregistry**
time="2020-07-15T14:31:28Z" level=info msg="serving registry" port=50051 type=appregistry
